R/splatPop-simulate.R

```diff
@@ -316,7 +394,9 @@ splatPopSimulateSC <- function(sim.means,
         assays(sim.all) <- sparsifyMatrices(assays(sim.all), auto = TRUE,
                                             verbose = verbose)
     }
-
+    
+    colnames(sim.all) <- paste(sim.all$Sample, sim.all$Cell, sep=":")
```

Comment:
I would probably have gone with just `SampleXCellY` without the `:` but happy to keep it. This should fix #118 right?